vscode/package.json

```diff
@@ -594,6 +594,13 @@
           "scope": "window",
           "order": 50
         },
+        "konveyor.kai.demoMode": {
+          "type": "boolean",
+          "default": false,
+          "description": "Whether to run Kai in demo mode i.e. use cached LLM responses",
+          "scope": "window",
+          "order": 50
+        },
```

Comment:
Can you not infer this from a non-null `konveyor.kai.cacheDir`? Why do we need both? Are we sure we should be sticking demo flags directly into our IDE config?

Reply:
@djzager there is a usecase where you would want the builtin cache (not setting cacheDir) to be used when in demo mode (setting demoMode). I don't see this being used by users that often, but for our demos we need this kind of thing.